<launch>
<node pkg="turtlesim" type="turtlesim_node" name="turtlesim_node"/>
<node pkg="assignment1b" type="square_closedloop.py" name="square_closedloop">
<param name="x" value="1"/>
<param name="y" value="1"/>
<param name="tol" value="0.1"/>
<param name="atol" value="0.01"/>
<param name="l_vel" value="0.2"/>
<param name="a_vel" value="0.2"/>
</node>
</launch>
